hypothesis-python/src/hypothesis/strategies/_internal/lazy.py

```diff
@@ -104,6 +104,9 @@ def calc_is_cacheable(self, recur: RecurT) -> bool:
                     return False
         return True
 
+    def calc_label(self) -> int:
+        return self.wrapped_strategy.label
+
     @property
```

Comment:
I don't intend for this to be an observable behavior change, I just wrote a testing script that called `s.calc_label()` instead of `s.label` and was confused why lazy strategy didn't forward the call.